backstop/tests/information-panel.html

```diff
@@ -725,98 +690,118 @@ <h1>Information panel</h1>
 
           <div class="iui-information-body">
 
-            <table class="demo-information-table">
-              <tr>
-                <td>Kind</td>
-                <td>MP3 audio</td>
-              </tr>
-              <tr>
-                <td>Size</td>
-                <td>9.7 MB</td>
-              </tr>
-              <tr>
-                <td>Where</td>
-                <td>/Shared/Music/</td>
-              </tr>
-              <tr>
-                <td>Date created</td>
-                <td>Aug 10, 2021 2:26 PM</td>
-              </tr>
-              <tr>
-                <td>Owner</td>
-                <td>
-                  <div class="demo-information-table-center">
-                    <user-icon
-                      size="small"
-                      type="2"
-                    >
-                    </user-icon>
-                    Terry Rivers
-                  </div>
-                </td>
-              </tr>
-              <tr>
-                <td>Last modified</td>
-                <td>Aug 10, 2021 3:26 PM</td>
-              </tr>
-              <tr>
-                <td>Modified by</td>
-                <td>
-                  <div class="demo-information-table-center">
-                    <user-icon
-                      size="small"
-                      type="1"
-                    >
-                    </user-icon>
-                    Robin Mercer
-                  </div>
-                </td>
-              </tr>
+            <div class="iui-information-body-content iui-inline">
+              <span class="iui-input-label">
+                Kind
+              </span>
+              <span>MP3 audio</span>
 
-            </table>
+              <span class="iui-input-label">
+                Size
+              </span>
+              <span>>9.7 MB</span>
```

Comment:
We have an extra `>` in here.
```suggestion
              <span>9.7 MB</span>
```

Reply:
You didn't see that 🤫